hack/deploy-cluster.sh

```diff
@@ -50,6 +66,122 @@ if ! [ -x "$(command -v aks-engine)" ]; then
   exit 1
 fi
 
+# initialize variables
```

Comment:
this added too much duplicated codes, please remove those lines.

Let's define a variable for `${REPO_ROOT}/examples/aks-engine.json`. When AZ is enabled, change the value to `${REPO_ROOT}/examples/az.json`

Reply:
Tried to do the same by defining a variable root_file=az.json